src/dxtbx/format/FormatCBFMiniEigerPetraP13.py

```diff
@@ -0,0 +1,38 @@
+"""An implementation of the CBF image reader for Eiger images"""
+
+
+from __future__ import annotations
+
+import sys
+
+from dxtbx.format.FormatCBFMiniEiger import FormatCBFMiniEiger
+
+
+class FormatCBFMiniEigerPetraP13(FormatCBFMiniEiger):
+    """A class for reading mini CBF format Eiger images, and correctly
+    constructing a model for the experiment from this. This tuned for Petra P13"""
+
+    @staticmethod
+    def understand(image_file):
+        """Check to see if this looks like an Eiger mini CBF format image,
+        i.e. we can make sense of it."""
+
+        header = FormatCBFMiniEiger.get_cbf_header(image_file)
+
+        for record in header.split("\n"):
+            if (
+                "# detector" in record.lower()
+                and "eiger" in record.lower()
+                and "E-32-0107" in record
+            ):
+                return True
+
+        return False
+
+    def _goniometer(self):
```

Comment:
This is functionally the same as the default constructor for goniometer which is implemented in the parent class?

Reply:
Yes, correct. Sorry, I did not check this carefully enough... Looks like we don't need any special code for P13 at all, it works out of the box once the serial number in FormatCBFMiniEigerPetraP14.py is fixed. So [b5e6cdd](https://github.com/cctbx/dxtbx/pull/626/commits/b5e6cddd66990487ff9505bba1c41267dc4e8057) is sufficient to make data from both P13 and P14 work.